BrowserKit/Sources/WebEngine/WKWebview/WKEngineInfo.swift

```diff
@@ -7,4 +7,6 @@ import Foundation
 /// The specific information related to the WK session and view
 struct WKEngineInfo {
     static var webserverPort = 6571
+
+    static let apostropheEncoded = "%27"
```

Comment:
Nit: we probably shouldn't hardcode these, since we can just use `String.addingPercentEncoding`. (Added a related comment below with a suggestion.)